Doc/cmus.txt

```diff
@@ -1230,6 +1230,12 @@ dsp.ao.wav_dir
 	Output directory for libao plugin, default is your home directory.
 	Used only if *dsp.ao.driver* is "wav".
 
+dsp.coreaudio.device
+	Device for Core Audio output. Leave empty for default.
+
+dsp.coreaudio.enable_hog_mode
+	Set true to set the device to hog mode. The default value is false.
```

Comment:
Hog mode is a CoreAudio feature that you gain direct access to the hardware. No other process can read from/write to the audio hardware. This feature is useful when you want to disable all other sound (such as system notification sound) send to the speaker. 

Audiophile players such as Vox or Foobar2000 have similar features.